### PR TITLE
Feature/teacher dash

### LINKED ISSUE #27: Create a teacherBody component
It must: 
 - [x] Hold a reading/instructions/TextArea Card Component
 - [x] Hold a QuestionCard Component
 - [x] Have a button for Preview/Generating Link

### LINKED ISSUE #28: Create a ReadingCard Component
It must: 
 - [x] Have a text Area

### LINKED ISSUE #29: Create A QuestionCard Component
It must: 
 - [x] Have a Question Input
 - [x] Have a correct Answer Input
 - [x] A Wrong Answer Input with plus and minus buttons next to it, follow the logic
 - [x] A new Question button
 - [ ] A delete question button, follow the logic
 - [ ] An add Reading button that renders a text area.

### PR DESCRIPTION
**Pull request by Melizzo**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures
## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 
## What is the current behavior?

- Kyle & created the Create Lesson Component that will allow a user to create a new lesson that will contain questions specific to that lesson.  Redux functionality was added upon submit of Lesson
## What is the new behavior?

-
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

We will need to add in the API call as soon as it's complete from the BE side. 
closes #29 
closes #27 
closes #28

### CODE REVIEW

File: src/CreateLesson/CreateLesson.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, {useState} from 'react'
+import './CreateLesson.scss'
+import QuestionFormCard from '../QuestionFormCard/QuestionFormCard'
+import LessonPreview from '../LessonPreview/LessonPreview'
+
+const CreateLesson = () => {
+  const [questions, setQuestions] = useState([])
+  const [lessonTitleText, setLessonTitleText] = useState("");
+
```
Comment (Melizzo) [inline src/CreateLesson/CreateLesson.js:9]: CreateLesson is the parent component that allows for the questions to be created & saved via hooks, and then to display them on the right side of the screen for a teacher.

File: src/LessonPreview/LessonPreview.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  setLesson,
+}) => {
+
+  const createLesson = (e) => {
+    e.preventDefault();
+    const lesson = {
+      name: lessonTitleText,
+      questions: questions,
+    };
+    setLesson(lesson);
```
Comment (Melizzo) [inline src/LessonPreview/LessonPreview.js:23]: Final functionality that sends the lesson object to Redux and to the BE once functionality is completed

File: src/LessonPreview/QuestionCard/QuestionCard.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './QuestionCard.scss'
+
+const QuestionCard = ({ id, question, allAnswers, reading, deleteQuestion})=> {
+  let correctAnswer = allAnswers.find(answer => answer.correct === true).desc
+  let incorrectAnswers = allAnswers
+    .filter(answer => answer.correct === false)
+    .map(answer => <p>{answer.desc}</p>)
```
Comment (Melizzo) [inline src/LessonPreview/QuestionCard/QuestionCard.js:8]: delete functionality for a teacher to remove a question from the lesson before submitting

File: README.md [hunk 4]
```diff
@@ -1,68 +1,35 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+# gumberoo-frontend
+ Frontend Tech:
```
Comment (Billwilke42) [inline README.md:2]: Thanks for adding to the README!

File: src/CreateLesson/CreateLesson.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import React, {useState} from 'react'
+import './CreateLesson.scss'
+import QuestionFormCard from '../QuestionFormCard/QuestionFormCard'
+import LessonPreview from '../LessonPreview/LessonPreview'
+
+const CreateLesson = () => {
+  const [questions, setQuestions] = useState([])
+  const [lessonTitleText, setLessonTitleText] = useState("");
+  
+  const deleteQuestion = (e) => {
```
Comment (Billwilke42) [inline src/CreateLesson/CreateLesson.js:10]: Really love that you made the form editable, great idea!!

File: src/LessonPreview/LessonPreview.js [hunk 6]
```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import "./LessonPreview.scss";
+import QuestionCard from "./QuestionCard/QuestionCard";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { setLesson } from "../actions";
+
+const LessonPreview = ({
```
Comment (Billwilke42) [inline src/LessonPreview/LessonPreview.js:8]: The way you both set up the preview is way better then it was in the MVP

File: src/LessonPreview/QuestionCard/QuestionCard.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './QuestionCard.scss'
+
+const QuestionCard = ({ id, question, allAnswers, reading, deleteQuestion})=> {
```
Comment (Billwilke42) [inline src/LessonPreview/QuestionCard/QuestionCard.js:4]: This is way better then the original MVP wireframe as well, great job!

File: src/TeacherLogin/TeacherLogin.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import './TeacherLogin.scss'
+
+const TeacherLogin = () => {
```
Comment (Billwilke42) [inline src/TeacherLogin/TeacherLogin.js:4]: Nice work on this even though we won't be using it in our demos

**Review by Billwilke42 — COMMENTED** [review]
This looks great, looking forward to pulling it to my branch. Next, we need to worry about routing and generating the link. Great progress!

**Pull request merged**